lib/src/models/curl.dart

```diff
@@ -34,6 +36,10 @@ class Curl extends Equatable {
   /// Sends data as a multipart/form-data request.
   final bool form;
 
+  /// Form data list.
+  /// Currently, it is represented as a list of key-value pairs ([key, value]).
+  final List<List<String>>? formData;
```

Comment:
It should be a `List<FormDataModel>`
See [here](https://github.com/foss42/apidash/blob/main/packages/apidash_core/lib/models/form_data_model.dart)
It will take care of files as well.

Reply:
I thought about doing that but didn't because U thought it will just add a dependency. Will surely do that now.

Reply:
No dependency must be added. You can create a copy of the model file ... It will be refactored later.